packages/react-data-grid/src/Row.js

```diff
@@ -138,7 +139,9 @@ const Row = React.createClass({
       return this.props.isSelected;
     } else if (typeof this.props.row.get === 'function') {
       val = this.props.row.get(key);
-    } else {
+    } else if (key.indexOf('.') >= 0) {
+      val = utils.getValueForKey(this.props.row, key);
```

Comment:
The signature for getValueForKey is `getValueForKey(key, model)`. Looks like it is being called with the wrong parameters here